code/game/objects/items/tanks/tanks.dm

```diff
@@ -44,37 +44,37 @@
 	var/list/explosion_info
 	/// List containing reactions happening inside our tank.
 	var/list/reaction_info
+	/// Mob that is currently breathing from the tank.
+	var/mob/living/carbon/breathing_mob = null
```

Comment:
what happens if the mob is deleted?

Reply:
That's a good point. I should change the tanks calls to it to `breathing_mob?.close_internals()` etc

Reply:
Fixed by commit 4ecc5a90b43978f60ecb50e92a76e0fa900015ad